include/aspect/melt.h

```diff
@@ -129,11 +129,24 @@ namespace aspect
     /**
      * Base class for material models that implement a melt fraction function.
      * This is used to compute some statistics about the melt fraction.
+     *
+     * This class is used as a "mix-in" class: Concrete material models will
+     * be derived from MaterialModel::Interface (and consequently have to
+     * implement the `virtual` functions of that class) *and also* from the
+     * current class (and consequently have to implement the `virtual` function
+     * of the current class). The inheritance from MaterialModel::Interface is
+     * then typically via the MaterialModel::MeltInterface intermediate class.
```

Comment:
I find the `then` in this sentence confusing. Nothing in the previous sentences seems to relate to it. I would find the current sentence easier to understand if the `then` would be omitted (`... is typically implemented via ...`